account_invoice_supplier_self_invoice/models/account_invoice.py

```diff
@@ -48,7 +48,14 @@ def create(self, vals):
                         )
         return super(AccountInvoice, self).create(vals)
 
-    @api.multi
-    def action_view_account_invoice_self(self):
+    @api.model
+    def action_view_account_invoice_self(self, docids, data=None):
         self.ensure_one()
```

Comment:
I think the ensure one is not necessary here

Reply:
Why do you add the @api.model here?